tests/dragonfly/requirements.txt

```diff
@@ -15,7 +15,7 @@ pytest-repeat==0.9.3
 pymemcache==4.0.0
 prometheus_client==0.17.0
 aiohttp==3.10.2
-numpy==1.24.3
+numpy
```

Comment:
Why this change?

Reply:
because numpy is better to install via apt and its version depends on each linux distribution.
these tests do not rely on a specific version